MAINTAINERS.md

```diff
@@ -3,7 +3,7 @@ The current Maintainers Group for the kpack Project consists of:
 | Name          | Employer | Responsibilities |
 |---------------|----------|------------------|
 | Tom Kennedy   | VMware   |                  |
-| Matthew McNew | VMware   |                  |
+| Matthew McNew |          |                  |
 | Daniel Chen   | VMware   |                  |
```

Comment:
@tomkennedy513 should @jjbustamante also be added here?

Reply:
I spoke with Juan and he said that he would like to contribute a bit more first